src/common/transformations/src/transformations/common_optimizations/fused_names_cleanup.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/common_optimizations/fused_names_cleanup.hpp"
+
+#include <openvino/cc/ngraph/itt.hpp>
+#include <transformations/rt_info/fused_names_attribute.hpp>
+
+bool ngraph::pass::FusedNamesCleanup::run_on_model(const std::shared_ptr<ngraph::Function>& f) {
+    RUN_ON_FUNCTION_SCOPE(FusedNamesCleanup);
+
+    bool is_graph_modified = false;
```

Comment:
I think graph is not modified, the graph structure is the same, the shapes are the same, no Validation pass needed